config/roles.yaml

```diff
@@ -7,17 +7,18 @@ roles:
         customize:
           - [modifyvm, !ruby/sym id, '--memory', 2048]
 
-  base_1024mb_ram:
+  base_4096mb_ram:
     providers:
       - type: virtualbox
         customize:
-          - [modifyvm, !ruby/sym id, '--memory', 1024]
+          - [modifyvm, !ruby/sym id, '--memory', 4096]
 
   master_base:
     synced_folders:
       - host_path:  'puppet'
         guest_path: '/tmp/puppet'
     provisioners:
+      - {type: shell, inline: "echo 'nameserver 8.8.8.8' > /etc/resolv.conf"}
```

Comment:
@smcclellan why'd this get added?

Reply:
I thought I needed that before to make it work. I'll nix the line and see if it still builds.